### ISSUE TITLE
Batch Norm Efficacy

### ISSUE THREAD

**Issue by Micky774** [Micky774/Generative-Models#22]
Run an experiment to compare the loss/stability of a batchnormed network vs the status quo

**Comment by AnonymousFool** [comment]
Experiment 18 (testing batch normalization for VtPVAE). Task file is exp18.sh in models.
